rust/cubestore/cubestore/src/bin/cubestored.rs

```diff
@@ -22,16 +22,21 @@ fn main() {
         .as_str()
         .unwrap()
         .to_string();
-    let metrics_mode = match std::env::var("CUBESTORE_METRICS") {
+    let metrics_format = match std::env::var("CUBESTORE_METRICS_FORMAT") {
         Ok(s) if s == "statsd" => metrics::Compatibility::StatsD,
         Ok(s) if s == "dogstatsd" => metrics::Compatibility::DogStatsD,
         Ok(s) => panic!(
-            "CUBESTORE_METRICS must be 'statsd' or 'dogstatsd', got '{}'",
+            "CUBESTORE_METRICS_FORMAT must be 'statsd' or 'dogstatsd', got '{}'",
             s
         ),
         Err(_) => metrics::Compatibility::StatsD,
     };
-    init_metrics("127.0.0.1:0", "127.0.0.1:8125", metrics_mode, vec![]);
+    let metrics_addr =
+        std::env::var("CUBESTORE_METRICS_ADDRESS").unwrap_or("127.0.0.1".to_string());
+    let metrics_port = std::env::var("CUBESTORE_METRICS_PORT").unwrap_or("8125".to_string());
+    let metrics_server_address = format!("{}:{}", metrics_addr, metrics_port);
+
+    init_metrics("127.0.0.1:0", metrics_server_address, metrics_format);
```

Comment:
```suggestion
    init_metrics("127.0.0.1:0", metrics_server_address, metrics_format, vec![]);
```

Reply:
thanks, @ovr I've done i blind merge now I've fixed it. build failed again I tried to understand